# marker_pose.srv
geometry_msgs/PoseStamped pose_msg
# Request
---
bool success 

